libevm/lib/service_evm.go

```diff
@@ -180,10 +193,31 @@ func (s *Service) EvmApply(params EvmParams) (error, *EvmResult) {
 		evmError = vmerr.Error()
 	}
 
+	traceLogs := geth_internal.FormatLogs(tracer.StructLogs())
+	if params.Trace == true {
+
+		if params.TxTraceParams.DisableStack == true {
```

Comment:
This is backwards, the parameters need to be set on the tracer, see the other comment above.